examples/langchain/Gemini_LangChain_QA_Chroma_WebLoad.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>I realise this wasn't you, but can you update "Gemini" to say "the Gemini API" here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Good catch Mark...I'll give notebooks a thorough read and make sure it's either saying "Gemini models" or "Gemini API".